pkg/mount/nfs.go

```diff
@@ -81,8 +81,12 @@ func (m *nfsMounter) normalizeSource(info *mount.Info, host string) {
 	if info.Fstype != "nfs" {
 		return
 	}
+
 	s := strings.Split(info.Source, ":")
 	if len(s) == 2 && len(s[0]) == 0 {
+		if strings.Contains(host, ":") && !strings.Contains(host, "[") {
+			host = fmt.Sprintf("[%s]", host)
```

Comment:
will adding the brackets here cause issues for existing NFS mounts? Thinking of the case where we upgrade PX and this change is added. Will PX adjust these mountpaths during a PX upgrade?

Reply:
No if the host has contains an ':' then it will be an IPv6 IP address.   If not then it will be dns entry and will not need it.

Reply:
Ah, makes sense :+1: